source/blink.cpp

```diff
@@ -284,8 +286,17 @@ void blink::application::run()
 					print("Finished compiling \"" + object_file.string() + "\" with code " + exit_code + ".");
 
 					// Only load the compiled module if compilation was successful
-					if (exit_code == "0")
-						link(object_file);
+					if (exit_code == "0") {
+						// Whatever happens while linking, we do not want to have the *.obj file in repository.
+						try {
+							link(object_file);
+						}
+						catch (...) {
+							DeleteFileA(object_file.string().c_str());
+							throw;
+						}
+						DeleteFileA(object_file.string().c_str());
```

Comment:
This should probably be called regardless of the exit code. There are cases where MSVC produces an object file despite an error I believe.

Reply:
Sure.